router-watchdog.js

```diff
@@ -58,11 +58,10 @@ function pingEndpoints() {
       if (failCounter >= CONFIG.numberOfFails) {
         print("Too many fails, resetting...");
         failCounter = 0;
-        Timer.clear(pingTimer);
         //set the output with toggling back
         Shelly.call(
           "Switch.Set",
-          { id: 0, on: true, toggle_after: CONFIG.toggleTime },
+          { id: 0, on: false, toggle_after: CONFIG.toggleTime },
```

Comment:
Should also be changed in the notification handler at the end of the file where the the state of the switch is monitored.

Reply:
done